routers/web/web.go

```diff
@@ -472,10 +472,12 @@ func RegisterRoutes(m *web.Route) {
 			m.Post("/delete", admin.DeleteRepo)
 		})
 
-		m.Group("/packages", func() {
-			m.Get("", admin.Packages)
-			m.Post("/delete", admin.DeletePackageVersion)
-		})
+		if setting.Packages.Enabled {
```

Comment:
Here it is fine as these routes are only called internally, and hence we can ensure using them correctly.